sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpEvent.kt

```diff
@@ -146,4 +159,14 @@ internal class SentryOkHttpEvent(private val hub: IHub, private val request: Req
         RESPONSE_BODY_EVENT -> eventSpans[CONNECTION_EVENT]
         else -> callRootSpan
     } ?: callRootSpan
+
+    fun scheduleFinish(timestamp: SentryDate) {
+        hub.options.executorService.schedule({
```

Comment:
I think this needs to be wrapped in a try/catch - in case the SDK / executor service is in the process of shutting down